# NormModel_SulcWidth_SZ
Normative, Network and Transcriptomic Modelling of Sulcal Width in Health and Schizophrenia

This repository contains information pertaining to the study Sulcal Widening in Schizophrenia Maps onto Sulcal Hubs and Energy-Synaptic Genes
(https://www.biorxiv.org/content/10.64898/2026.01.09.698648v1). The docker for sulci-processing, the corresponding sulcal parcellation in 'fsaverage' space as well as the code for minimalistic sulci visualisation were adapted from [Snyder et al. 2024](https://www.cell.com/neuron/article/S0896-6273(24)00568-3/fulltext). For normative modelling we created a docker that includes the [PCN-toolkit](https://pcntoolkit.readthedocs.io/en/latest/) and added extra automated features for model evaluation and visualization. Instructions for installing and running the docker for normative modelling can be found [here](https://github.com/iamjoostjanssen/NormModel_MorphoSim_SZ/blob/main/Docker_and_ReferenceModelling.txt). The reference sulcal-to-sulcal connectome was built using data from the [HCP](https://www.humanconnectome.org/). All but one sample used in the study are open source and the repositories are mentioned in the manuscript. Spatial correlations were corrected by spin tests using code from [Váša et al. 2018](https://academic.oup.com/cercor/article/28/1/281/4566607?login=false). The Q-Q plots for all included sulci can be found [here](https://github.com/iamjoostjanssen/NormModel_Sulci/tree/main/Q-Q%20plots). All tools used for the transcriptomic analyses are open source and the tools are referenced in the manuscript. Here's a visual summary of the methods used in the study:
<img width="1115" height="720" alt="graphical_summary_methods_inverted" src="https://github.com/user-attachments/assets/c2b5b285-38ea-4390-8292-0a581ef43c88" />
Top row left to right: Sulcal width was measured for 40 bilateral sulci in 5,392 individuals. Normative modeling provided z-scores, representing individual deviations from the normative sulcal width range, which were used in diagnostic group comparisons between individuals with SZ and healthy individuals, producing a vector a t-statistics. Middle row left to right: Reference cortico-cortical sulcal connectivity was estimated from diffusion-weighted imaging (DWI) in an independent dataset of 100 healthy individuals. Reference degree centrality was estimated for each sulcus in a first step; in a second step, the reference degree-centrality vector was correlated with the vector of diagnostic group differences (t-statistics). Bottom row right to left: Partial least squares regression (PLS) was used to assess the relationship between the vector of t-statistics for the left hemisphere sulci and a sulcal gene expression matrix comprising the expression levels of 15,633 genes for each left hemispheric sulcus. PLS analysis identified genes positively and negatively associated with the t-statistics. Finally, gene enrichment analyses characterized the biological processes and transcriptional underpinnings linked to the diagnostic group differences.
